jkind-api/src/jkind/api/JKindApi.java

```diff
@@ -216,7 +225,16 @@ private ProcessBuilder getJKindProcessBuilder(File lustreFile) {
 	}
 
 	protected String[] getJKindCommand() {
-		return new String[] { ApiUtil.getJavaPath(), "-jar", getOrFindJKindJar(), "-jkind" };
+		List<String> args = new ArrayList<>();
+		args.add(ApiUtil.getJavaPath());
+		args.addAll(vmArgs);
+		args.add("-jar");
+		args.add(getOrFindJKindJar());
+		args.add("-jkind");
+		
+		String[] rv = new String[args.size()];
+		rv = args.toArray(rv);
+		return rv;
```

Comment:
You can get rid of the extra `rv` variable by just doing `args.toArray(new String[args.size()])`